src/main/java/com/salesforce/functions/jvm/sdk/data/bulk/BulkJobState.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+package com.salesforce.functions.jvm.sdk.data.bulk;
+
+public enum BulkJobState {
```

Comment:
missing `Open`

Reply:
Interesting, I've assumed both query and ingest share the same state enum, but they don't. I did assume so, because query has "UploadComplete" which does not make much sense to me.